src/Modulo2.jl

```diff
@@ -271,6 +272,21 @@ function copyto!(a::ZZ2Array, b::ZZ2Array)
     end
 end
 
+"""
+    copyto!(a::ZZ2Vector, num::Integer) 
+
+Copy the bit representation of number `num` into ZZ2Vector `a`
+"""
+function copyto!(a::ZZ2Vector, num::T) where T <: Integer
+    !isbits(num) && throw(ArgumentError("Argument num is not a bits type (BigInt not supported)!"))
```

Comment:
I'm not sure that `isbits` is a good criterion. One could cook up a new integer type that satisfies this, yet has no meaningful bit size. What about the type restriction
```
T <: Union{Base.BitInteger,BitIntegers.AbstractBitSigned,BitIntegers.AbstractBitUnsigned}
```
Also, one needs to test that `a` can hold the result, and one may have to adjust its length.

Reply:
Sure. My (perhaps too gung-ho) approach was to add functionality incrementally instead making an iron-clad implementation right off the bat. As far as length adjustment: since `ZZ2Vector` does not support `push!` etc., I thought that for the moment it's best to consider its length fixed, and if it does not fit the type, through an error. I assumed here that all the `isbits` types are smaller than 256 so that would not be an issue. There is a separate matter of what if `length(a)` is not the same as the bit size of `num`, even if its internal buffer can fit it. Thoughts?